Sources/BraintreeThreeDSecure/BTThreeDSecureClient.swift

```diff
@@ -59,7 +59,7 @@ import BraintreeCore
                 return
             }
 
-            if request.amount?.decimalValue.isNaN == true || request.amount == nil {
+            if request.amount.isEmpty {
                 NSLog("%@ BTThreeDSecureRequest amount can not be nil or NaN.", BTLogLevelDescription.string(for: .critical))
                 let error = BTThreeDSecureError.configuration("BTThreeDSecureRequest amount can not be nil or NaN.")
```

Comment:
Lets update these strings to accurately reflect this block now that amount cannot be nil or NaN. We can instead update to something like "BTThreeDSecureRequest amount cannot be an empty string"